src/core/link.js

```diff
@@ -205,7 +205,7 @@
          * @return {String} The URI updated with the protocol.
          */
         _getCompleteURI: function(URI) {
-            if (URI.indexOf('@') && URI.match(/[^\/]+@/)) {
+            if (URI.indexOf('@') && URI.test(/[^\/]+@/)) {
```

Comment:
AlloyEditor does not accept commit messages like `.test instead of .match, duh`.
The message should make sense and the issue number should be specified in the commit.

Thanks,